src/app/file-manager/services/firecloud-api-mock.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { FirecloudService } from './firecloud.service';
+import { Observable } from 'rxjs/Observable';
+import * as workspaces from 'assets/demo/workspaces';
+
+@Injectable()
+export class FirecloudApiMockService implements FirecloudService {
+
+
+  constructor(private http: HttpClient) { }
+
+
+  public getUserFirecloudWorkspaces(optional: boolean) {
+    return Observable.of(workspaces.default.content())
+      .map((resp: any) => {
+        const workspacesList: any[] = new Array();
+        resp.forEach(item => {
+          const workspace = FirecloudService.getWorkspaceData(item, optional);
```

Comment:
Since this inherits from `FirecloudService` seems like you could call `getWorkspaceData()` directly, e.g.
```
const workspace = getWorkspaceData(item, optional);
```